### ISSUE TITLE
Razer naga Pro

### ISSUE THREAD

**Issue by MrMFD** [1kc/razer-macos#629]
First of all I would like to say thank you for everything that you all have done to make this possible !!! you are all heroes !! 

I use the naga pro wirelessly and I would just like to comment in one post what is still missing...it would be great if it could be added (**,) 

Features missing:
1. USB not working 
2. No battery level 
3. RGB not 100% working ( only spectrum works 100% )
   3.1 When selecting another profile, only the Razer logo and the scroll wheel effect the changes. For example, if red static is chosen, the scroll wheel and Razer logo maintains the red while the side buttons carry on with spectrum for eg.
   3.2 When OFF is selected, only scroll wheel and Razer Logo switches off, the side buttons maintain displaying colour.
4. Brightness is missing functionality.
5. DPI is missing functionality.
6. Cannot verify if polling rate works
I just want to say a huge thank you to you all !! just being able to use this on Mac is fabulous and that is all thanks to the contributors.

**Comment by aitaDev** [comment]
^ also not being detected by the app while connected via bluetooth. although the mouse works with the native OS

**Comment by qauff** [comment]
Would love to have bluetooth support for Naga Pro also

**Comment by skc18** [comment]
Yes, please support bluetooth naga pro. Using it mainly through bluetooth

**Comment by LJ-03** [comment]
Would like to also ask for bluetooth support for the Naga pro wireless
